### DISCUSSION TITLE
Never combine with no label

### DISCUSSION THREAD

**Discussion by kakapurinaveen** [valinet/ExplorerPatcher#65] [Q&A] [answered]
Is there any modification to allow taskbar have icons as never combine but with no label (text), currently the never combine feature shows text along with the icon takes a lot of space.

**Comment by valinet** [comment, marked as answer]
From https://github.com/valinet/ExplorerPatcher/issues/9:

Download the beta version of 7+ TaskbarTweaker. Then, follow this:

> [...] the beta version on the web site works just fine for me and does not ask for any code, and I was able to test what he suggested as well. First of all, yeah, set taskbar to "Always combine, hide labels", then it 7TT choose "Don't combine grouped buttons" in "Combining" and even "Don't group" in "Grouping" and then it is basically exactly what you asked for. Yeah, counterintuitive, but it works. Thanks @Gaurav-Original-ClassicShellTester.
>
> ![image](https://user-images.githubusercontent.com/6503598/136698823-9b6df8bf-9919-484d-9f67-08ad3cc1664d.png)
